docs/project/contribute.md

```diff
@@ -13,6 +13,9 @@ Pull Requests welcome for any level of improvement, from a small typo to a new s
 
 All the source code is available at [github.com/all-contributors/all-contributors](https://github.com/all-contributors/all-contributors/tree/master/docs)
 
+## New Contributor Onboarding
+We offer new contributor onboarding, if you're interested in taking part in this please comment in the slack channel. 
```

Comment:
It might be better to link the slack channel here even tho people can find it in the README but some seem to miss that part.